tests/formatter_tests.ts

```diff
@@ -62,16 +62,10 @@ const FormatterTests = {
 /** Calculate the glyph's width in the current music font. */
 // How is this different from Glyph.getWidth()? The numbers don't match up.
 function getGlyphWidth(glyphName: string): number {
-  // `38` seems to be the `fontScale` specified in many classes, such as
-  // Accidental, Articulation, Ornament, Strokes. Does this mean `38pt`???
-  //
-  // However, tables.ts specifies:
-  //   NOTATION_FONT_SCALE: 39,
-  //   TABLATURE_FONT_SCALE: 39,
-  const musicFont = Tables.currentMusicFont();
-  const glyph: FontGlyph = musicFont.getGlyphs()[glyphName];
-  const widthInEm = (glyph.xMax - glyph.xMin) / musicFont.getResolution();
-  return widthInEm * 38 * Font.scaleToPxFrom.pt;
+  const el = new Element();
+  el.setText(String.fromCharCode(parseInt(glyphName, 16)));
```

Comment:
We should default to using String.fromCodePoint() instead, since we are dealing with large unicode code points in our library. Some of the SMuFL codes (e.g., alternates) exceed the limit of fromCharCode().

... Unless we find a performance critical loop where fromCharCode() would help.

Reply:
reworked